packages/esm-patient-search-app/src/patient-search-page/patient-search-lg.component.tsx

```diff
@@ -96,6 +95,8 @@ const PatientSearchComponent: React.FC<PatientSearchComponentProps> = ({
             totalPages={totalPages}
           />
         </div>
+      ) : (
+        <div className={styles.spacer} />
```

Comment:
So the end of the results container isn't flush with the end of the page. This gives the container the same amount of padding as is applied when pagination is applied. So the spacing remains consistent in either case.